### ISSUE TITLE
Send typing notice in the background

### ISSUE THREAD

**Issue by ulyssa** [ulyssa/iamb#96]
I suspect that sending the typing notice update is what's causing some of the freezes that people in `#iamb-users` are observing when their network is slow or cuts out. This can probably be moved into a `tokio::spawn` to avoid waiting for it to complete.

**Comment by VAWVAW** [comment]
Are we sure that this is a problem? The sync part just sends a message to a channel, the store is never locked and the sdk debounces the typing notice requests internally.
I don't see how we can make the more performant and I think people who run into this should just set `settings.typing_notice_send = false`.

**Comment by ulyssa** [comment]
Yeah, I'm convinced that the issue was likely #99 or something similar, where the worker was holding onto the mutex for the shared store for too long. Enough has changed in the worker and I've never actually observed this when typing myself, so I'm fine closing this out as not a bug.
